OpenRA.Mods.Common/Widgets/Logic/Editor/SaveMapLogic.cs

```diff
@@ -190,21 +202,21 @@ public SaveDirectory(Folder folder, string displayName, MapClassification classi
 
 					map.Save(package);
 
-					Console.WriteLine("Saved current map at {0}", combinedPath);
+					Console.WriteLine($"Saved current map at {combinedPath}");
 					Ui.CloseWindow();
 
 					onSave(map.Uid);
 				}
 				catch (Exception e)
 				{
-					Log.Write("debug", "Failed to save map at {0}: {1}", combinedPath, e.Message);
-					Log.Write("debug", "{0}", e.StackTrace);
+					Log.Write("debug", $"Failed to save map at {combinedPath}: {e.Message}");
+					Log.Write("debug", $"{e.StackTrace}");
 
 					ConfirmationDialogs.ButtonPrompt(
-						title: "Failed to save map",
-						text: "See debug.log for details.",
+						title: modData.Translation.GetFormattedMessage(SaveMapFailedTitle),
```

Comment:
Having every caller use `GetFormattedMessage` for every element is a *lot* of duplication. Can we move these inside the `ButtonPrompt`?

Reply:
Would work here, but not with arguments.
```csharp
title: modData.Translation.GetFormattedMessage(ResetTitle, Translation.Arguments("panel", panels[activePanel])),
```

Reply:
Add arguments to `ButtonPrompt` to pass in the arguments object.